models/articles.model.js

```diff
@@ -29,3 +29,21 @@ exports.fetchArticleById = (article_id) => {
         return res.rows[0]
     })
 }
+
+
+exports.updateArticleById = (article_id, newVote) => {
+    if(isNaN(article_id) || isNaN(newVote)) {
+        return Promise.reject({status: 400, msg: "Bad Request"})
```

Comment:
A PSQL error will be thrown if article_id is not a number, so we don't need this bit of code 👍